publishing-bot/manifests/rs.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apps/v1
+kind: Replicaset
```

Comment:
Why is this not using Deployment?

Reply:
@nikhita asked to keep it as Replicaset in a previous conversation: https://github.com/kubernetes/k8s.io/pull/520#discussion_r368516238

Reply:
I don't see why this is needed, though.  Is it JUSt to avoid changing 2 things at once?